Support setting profiles-dir (#25)
godatadriven/pytest-dbt-core: CHANGELOG.md

- Add example with column rename macro ([PR](https://github.com/godatadriven/pytest-dbt-core/pull/19))

> ## [0.1.1] - 2023-02-22

Reviewer: You can remove this and add your entry on line two. The release script automatically adds the version number and date section header

Author: changed https://github.com/godatadriven/pytest-dbt-core/pull/25/commits/9c04c9242202d3573ee12a37a566f7a3bc60fcea

> - Expore `--profiles-dir` as option for setting profiles directory instead of using `DBT_PROFILES_DIR` environment variable

Reviewer: ```suggestion
- Add `--profiles-dir` as an option for setting the profiles directory
```
